package.json

```diff
@@ -5,9 +5,11 @@
 	"license": "MPL-2.0",
 	"scripts": {
 		"lint": "prettier --write . && eslint --fix .",
-		"lint:ci": "eslint .",
+		"lint:ci": "prettier --check . && eslint .",
 		"inc": "tsc -p ./util/configs/tsconfig.massVer.json && node ./util/tools/massVer.js",
-		"se": "tsc -p ./util/configs/tsconfig.auto.syntaxEnforcer.json && node ./util/tools/auto/syntaxEnforcer.js",
+		"incChanged": "tsc -p ./util/configs/tsconfig.auto.syntaxEnforcer.json && node ./util/tools/bumpChanged",
```

Comment:
```suggestion
		"bumpChanged": "tsc -p ./util/configs/tsconfig.auto.syntaxEnforcer.json && node ./util/tools/bumpChanged",
```

Reply:
No bc consistency

Reply:
I would either change the file name or the package script name. So they are the same like all other scripts.

Reply:
For the same reason that massVer is called inc in the script, this is called incChanged as it does the same thing to the files in the diff.

Reply:
Well while you are at it you can make the scripts and file names consistent

Reply:
That’s unnecessary, inc and incChanged are much easier to understand that two letter acronyms